src/main/drivers/timer_hal.c

```diff
@@ -784,6 +815,39 @@ static void timCCxHandler(TIM_TypeDef *tim, timerConfig_t *timerConfig)
 #endif
 }
 
+static inline void timUpdateHandler(TIM_TypeDef *tim, timerConfig_t *timerConfig)
+{
+    uint16_t capture;
+    unsigned tim_status;
```

Comment:
Preferable to give the size rather than just state unsigned.

Reply:
I kept it as is as tim_status is generally kept as unsigned in other functions defined in this file as well. Compiler should have no issues with optimizing it.